package.json

```diff
@@ -1,30 +1,31 @@
 {
   "name": "@govuk-react/monorepo",
   "scripts": {
-    "start": "cd packages/storybook && npm start",
     "bootstrap": "lerna bootstrap",
-    "docs": "yarn docs:components && concat ./components/*/README.md -o './API.md'",
+    "build:dist": "yarn build:es && rimraf dist && webpack -p packages/govuk-react/es/index.js --devtool source-map --output-filename index.js",
+    "build:es": "lerna run build:es --parallel",
+    "build:lib": "lerna run build:lib --parallel",
+    "build:storybook": "cd packages/storybook && yarn build",
+    "build:sync": "lerna run build",
+    "build": "lerna run build --parallel",
+    "bundlesize": "bundlesize",
+    "clean:workspace": "rimraf packages/*/lib && rimraf packages/*/es && rimraf components/*/lib && rimraf components/*/es",
+    "clean": "rimraf lib && rimraf es",
+    "coverage:visual-regression": "jest --watchAll=false --coverage --coverageDirectory=coverage/visual-regression --roots=\"<rootDir>\" --testMatch=\"<rootDir>/storyshots/index.js\" --config=\"./jest.storyshots.config.js\"",
```

Comment:
this is the only additional command

Reply:
annoyingly, only stories that are enabled in chromatic actually constitute coverage...

do you think we can/should make this calculate accordingly?

side note, I should reenable chromatic...